Permission updates (#685)
CloudCannon/platform-documentation: developer/articles/what-are-branch-defaults.mdx

By default, *Team Members* in *Custom Permission Groups* can always view, edit, or merge the branched *Sites* they create. CloudCannon does this by granting new permissions to their *Custom Permission Group* when they branch a *Site* inside a *Project*.

> CloudCannon adds these permissions to each *Custom Permission Group* that gave the *Team Member* permission to create the branched *Site*. A group qualifies when it has the `site-branch:create` or `*:create` resource with a Global scope, a scope targeting the *Site* being branched, or a scope targeting the *Project* containing that *Site*. Groups with branching permissions scoped to a different *Project* or *Site*, groups where the branching resource is an exception, and groups without any branching permission are unaffected.

Reviewer: ```suggestion
CloudCannon adds these permissions to each *Custom Permission Group* that gave the *Team Member* permission to create the branched *Site*. A group qualifies when it has the `site-branch:create` or `*:create` resource with a Global scope, or a *Site* or *Project* scope including the branched *Site*. Other *Permission Groups* are not updated, such as those scoped to a different *Project* or *Site*, groups where the branching resource is an exception, and groups without any branching permissions.
```
